Builds/levelization/README.md

```diff
@@ -0,0 +1,74 @@
+# Levelization
+
+Levelization is the term used to describe efforts to prevent rippled from
+having or creating cyclic dependencies.
+
+rippled code is organized into directories under `src/rippled` (and
+`src/test`) representing modules. The modules are intended to be
+organized into "tiers" or "levels" such that a module from one level can
+only include code from the same or lower levels, though including code
```

Comment:
Permitting edges at the same level permits the possibility of a cycle. If there are no cycles in practice, then it is possible to introduce new levels as needed such that no module depends on another module at the same level. Should we just change the permission and introduce those levels?

Reply:
Yes, you're right. I'll reword this.